src/main/java/Duke.java

```diff
@@ -23,72 +23,104 @@ public static void main(String[] args) {
             String arr[] = input.split(" ", 2);
             //Task t = new Task(input);
 
-            if (!input.equals("bye")) {
-                if (input.equals("list")) {
-                    System.out.println("    -----------------------------------\n"
-                            + "      Here are the tasks in your list:");
-                    for (int i = 1; i < taskList.size() + 1; i++) {
-                        Task current = taskList.get(i-1);
-                        System.out.println("      " + i + ". " + current);
-                    }
-                    System.out.println("    -----------------------------------");
+            if (input.equals("bye")) {
+                System.out.println("    -------------------------------------------------\n"
+                        + "      Bye. Hope to see you again soon! :)\n"
+                        + "    -------------------------------------------------");
```

Comment:
Since the horizontal line as extensively used in Duke, perhaps you can create a final static String variable LINE. This can help to make the code cleaner! :-)